### PR TITLE
Create review controller view

### PR DESCRIPTION
**Pull request by coderJules**

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -2,6 +2,7 @@ class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
   before_action :authenticate_user!
 
+
```
Comment (OKMantis) [inline app/controllers/application_controller.rb]: Could you take away on line so that there's only 1 empty line above the text.

File: app/policies/review_policy.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+  def new?
+   true
+  end
+
+  def create?
+    true
+  end
+
+  # def update?
```
Comment (OKMantis) [inline app/policies/review_policy.rb]: Could you take away the commented stuff. We don't need it.

File: app/views/experiences/show.html.erb [hunk 3]
```diff
@@ -2,10 +2,26 @@
...
+  <p> $<%= @experience.price %></p>
+  <div style="background-color: lightgrey;">
+  <p><strong>Reviews</strong></p>
+  <% @experience.reviews.each do |review| %>
+  <% reviewy = review.rating %>
+
+  <% review.rating.times do |n| %>
+    <i class="fa fa-star"></i>
+  <% end %>
+  <%# ( 5 - review.rating ).times do %>
```
Comment (OKMantis) [inline app/views/experiences/show.html.erb]: Is this commented line still needed? If not, please take it away.

File: app/views/reviews/create.html.erb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+<h1>Reviews#create</h1>
+<p>Find me in app/views/reviews/create.html.erb</p>
```
Comment (OKMantis) [inline app/views/reviews/create.html.erb]: Why did you add a view file for create? What is supposed to be shown here? I think it's not necessary, unless you have a reason.

File: app/views/reviews/new.html.erb [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+        <%= @review.errors.full_messages.each do |message| %>
+          <li><%= message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <%= f.input :rating %>
+  <%= f.input :description %>
+  <%= f.submit "add a review", class: "btn btn-primary" %>
```
Comment (OKMantis) [inline app/views/reviews/new.html.erb]: In app/views/experiences/shared/_form.html.erb I've made a form which I've used for experience/new. Maybe it's good idea if you do the same for your form.

File: config/routes.rb [hunk 6]
```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
+
   root to: 'experiences#index'
 
+
```
Comment (OKMantis) [inline config/routes.rb]: Too many empty lines, please limit it to 1 empty line.

File: test/controllers/reviews_controller_test.rb [hunk 7]
```diff
@@ -0,0 +1,14 @@
+require 'test_helper'
```
Comment (OKMantis) [inline test/controllers/reviews_controller_test.rb]: What is this controller for? Do we need it?

File: test/policies/review_policy_test.rb [hunk 8]
```diff
@@ -0,0 +1,19 @@
+require 'test_helper'
```
Comment (OKMantis) [inline test/policies/review_policy_test.rb]: Is this also a test file? Do we need? If not, let's delete it.

File: test/controllers/reviews_controller_test.rb (hunk 7 above)
Comment (coderJules) [reply test/controllers/reviews_controller_test.rb]: Hey Owen, I think these must be automatically generated? I don't know if they're needed, but it looks like there is one for every controller.

File: app/views/reviews/new.html.erb (hunk 5 above)
Comment (coderJules) [reply app/views/reviews/new.html.erb]: Hey, I think this is a good idea if you have both a new and update page. Do you think I need to separate them if there is only one form using it?

File: test/policies/review_policy_test.rb (hunk 8 above)
Comment (coderJules) [reply test/policies/review_policy_test.rb]: As with the other test file, I think these are also automatically created. I think if we didn't want this we maybe need to add -T when we create controllers to removes the tests. That is just my guess though

File: app/views/reviews/new.html.erb (hunk 5 above)
Comment (OKMantis) [reply app/views/reviews/new.html.erb]: Ok I understand. It's a bit redundant. We'll leave it.

File: test/controllers/reviews_controller_test.rb (hunk 7 above)
Comment (OKMantis) [reply test/controllers/reviews_controller_test.rb]: Ok, we'll leave it the way they are.

File: test/policies/review_policy_test.rb (hunk 8 above)
Comment (OKMantis) [reply test/policies/review_policy_test.rb]: Ok, approved.
